eduard.iskandaryan/repo/Lesson_3/exercise_03_13/exercise_03_13.py

```diff
@@ -0,0 +1,5 @@
+meal_cost = float(input("Please enter the cost of the meal: "))
+meal_tax = meal_cost * 10/100
```

Comment:
CSW: Space before and after math operator.
LW: Use constant variable for tax percent value.